3.Tests/UI/StepDefinitions/Settings/Account/ViewStepDefition.cs

```diff
@@ -0,0 +1,39 @@
+﻿using SeleniumExtras.WaitHelpers;
+using TechTalk.SpecFlow;
+using Todoly.Core.UIElements.Drivers;
+using Todoly.Core.UIElements.WebActions;
+using Todoly.Tests.UI.Steps.Commons;
+using Todoly.Views.WebAppPages;
+
+namespace Todoly.Tests.UI.Steps.RecycleBin;
+
+[Binding]
+[Scope(Feature = "Settings account view")]
+public class AccountViewStepDefinition : CommonSteps
+{
+    private readonly ScenarioContext _scenarioContext;
+
+    private readonly HomePage _homePage;
+
+    public AccountViewStepDefinition(ScenarioContext scenarioContext) : base(scenarioContext)
+    {
+        _scenarioContext = scenarioContext;
+        _homePage = new HomePage();
+    }
+
+    [When(@"the user clicks on the Settings option and then account tab")]
+    public void WhentheuserclicksontheSettingsoptionandthenaccounttab()
+    {
+        _homePage.SettingsButton.Click();
```

Comment:
There is starting to be a lot of duplicated code, ViewProfileStepDefinition has a step to click on the SettingsButton already, we need to evaluate this.